pylinac/winston_lutz.py

```diff
@@ -366,6 +366,10 @@ def __init__(self, file):
         self.check_inversion()
         # Flip the image upside down... makes it more intuitive later on.
         self.flipud()
+        # The first pixel line on the left has a glitch on Elekta iview EPID
```

Comment:
Do you have a sample dataset to show this effect? And how confident are you that this is true for all Elektas?